NitroxPatcher/Patches/Persistent/PAXTerrainController_LoadAsync_Patch.cs

```diff
@@ -36,8 +37,8 @@ public static IEnumerable<CodeInstruction> Transpiler(MethodBase original, IEnum
                    {
                         //Call our injection code and set paxTerratinController.setIsWorking(true); (this keeps up the loading screen)
                         yield return new ValidatedCodeInstruction(OpCodes.Call, typeof(Multiplayer).GetMethod(nameof(Multiplayer.SubnauticaLoadingCompleted), BindingFlags.Public | BindingFlags.Static));
-                        yield return new ValidatedCodeInstruction(OpCodes.Ldc_I4_1, instruction.labels);
-                   }
+                        yield return new CodeInstruction(OpCodes.Ldc_I4_1) { labels = instruction.labels };
```

Comment:
Why can we no longer use this in conjunction with ValidatedCodeInstruction?

Reply:
The ValidatedCodeInstruction when attempting to pass a null Operand with the set label fails to run properly, and passing the labels as an operand does not agree well with Harmony